### DISCUSSION TITLE
I struggled with this docs and with how it's supposed to work.

### DISCUSSION THREAD

**Discussion by nooksmr** [dscalzi/Nebula#63] [Q&A] [answered]
I was looking to make a small modded Launcher (client mods only), already very unhappy that Fabric isn't supported for both Helios and Nebula, I couldn't get the program to work.

I'm not a beginner, I know how to read documentation, but there's none, except for a list of commands and a few hints here and there.

I could use some help. Incidentally, I can't put in a recommended forge version 1.19.4, it's marked as "null". I don't understand that either. I'm doing everything correctly.

All this to put 3-15 mods and make it easier for people to connect...

I'm struggling. Anyone can help me please ? DM @alzhemp4 in Discord, Twitter or here with my actual account.

**Comment by sjempotje** [comment, marked as answer]
Hey @alzhetv !

Welcome! Would you mind joining the Aventium Softworks Discord? https://discord.gg/AcsAjrMpd5 . Support is given way faster there than on github (most of the time).

Greetings,
Bart Hazewinkel (sjempotje)

**Reply by nooksmr** [reply]
Hi. Joined it today, and therefor one member did tell me the exact same thing about hours ago, thanks to him.
-> By the way, the solution that isn't really wrote in readme helped me A LOT on where to do things.

Definitively the TL;DR guide is very fine. Thoses particular steps helped a lot.
6.
 Put your mods, and libraries in folder (aka actual HeliosLauncher sourcecode) ROOT\servers\yourserver-mcversion

8. Upload generated files in your web server <--- This was also appreciated.
9. On your launcher, in app/assets/js/distromanager.js on line 7, edit the url to match your own distribution.json url <--- and this ofc.
